src/webparts/projectFilesViewer/components/ProjectFilesViewer.tsx

```diff
@@ -1,30 +1,37 @@
 import * as React from 'react';
-import styles from './ProjectFilesViewer.module.scss';
-import { IProjectFilesViewerProps } from './IProjectFilesViewerProps';
+import { IProjectFilesViewerProps } from './models/IProjectFilesViewerProps';
+import { State} from './models/IProjectFilesViewerState';
```

Comment:
Change the name from `State` to `IProjectFilesViewerState`

Reply:
Done